donkeycar/parts/throttle_filter.py

```diff
@@ -10,15 +10,16 @@ def __init__(self):
 
     def run(self, throttle_in):
         throttle_out = throttle_in
+        # In case pilot's throttle and angle is None type
+        if throttle_in != None:
```

Comment:
This shouldn't be checked here, `throttle_in` is expected to be a number. Do you see a `None` being produced by the pilot?

Reply:
As it seems the pilot throttle is `NoneType` at start until I switch the drive mode to local pilot or local angle, then it will become `numpy.float32` afterward.

Here is the result if I remove the checking:
<img width="504" alt="terminal output" src="https://user-images.githubusercontent.com/38518793/126586835-f2cdc149-9710-42fc-9106-3b7a2cbb9d9f.png">